src/Model/Setting/InvoiceDefaults.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Model\Setting;
+
+use Billogram\Model\CreatableFromArray;
+
+class InvoiceDefaults implements CreatableFromArray
```

Comment:
I see. But why is it named "InvoiceDefaults"? It makes me believe that there are some other kind of invoice type...